arch/xtensa/src/esp32s3/esp32s3_dma.h

```diff
@@ -64,7 +64,7 @@ extern "C"
 
 /* DMA channel number */
 
-#define ESP32S3_DMA_CHAN_MAX          (3)
```

Comment:
@acassis , do you recall why didn't you assign all 5 GDMA channels to S3? I looked for uses without using the `esp32s3_dma_request` "API" and I couldn't fin anything using GDMA channels directly.